utilities/game.py

```diff
@@ -19,3 +20,25 @@ def is_red() -> bool:
 
 def get_team() -> wpilib.DriverStation.Alliance:
     return wpilib.DriverStation.getAlliance()
+
+
+def pose3d_to_translation2d(pose3d: Pose3d) -> Translation2d:
+    return Translation2d(pose3d.x, pose3d.y)
+
+
+def find_closest_tag(robot_pose: Pose2d) -> tuple[Pose3d, int]:
+    tag_ids = [1, 2, 3] if is_red() else [6, 7, 8]
+
+    # Use the first tag to set a baseline for distance
+    best_id = tag_ids[0]
+    best_tag = apriltag_layout.getTagPose(best_id)
+    closest_distance = robot_pose.translation().distance(pose3d_to_translation2d(best_tag))  # type: ignore
```

Comment:
I presume the type check failure is because of the optional?
```suggestion
    assert best_tag is not None
    closest_distance = robot_pose.translation().distance(pose3d_to_translation2d(best_tag))
```
Alternatively we could get all of the tags in this module and assert they're all not None, like we're already doing for a couple of them.

Reply:
lmao we both basically did this at the same time